portal/migrations/versions/38cfbd643c73_.py

```diff
@@ -26,49 +26,41 @@ def upgrade():
     bind = op.get_bind()
     session = Session(bind=bind)
 
-    # get existing Observation.audit_id values
-    audit_ids = {}
-    for obs_id, audit_id in session.execute('SELECT id, audit_id FROM observations'):
-        audit_ids[obs_id] = audit_id
-
     op.drop_constraint(u'observations_audit_id_fkey', 'observations', type_='foreignkey')
     op.drop_column('observations', 'audit_id')
     op.add_column('user_observations', sa.Column('audit_id', sa.Integer(), nullable=True))
     op.create_foreign_key(u'user_observations_audit_id_fkey', 'user_observations',
                           'audit', ['audit_id'], ['id'])
 
-    # store existing audit_id values in UserObservations, based on observation_id
-    for obs_id in audit_ids:
+    # create new audits for UserObservations
+    for uo_id, user_id in session.execute('SELECT id, user_id FROM user_observations'):
+        aud = Audit(user_id=user_id, subject_id=user_id,
+                    comment="generated as part of migration 38cfbd643c73 upgrade")
```

Comment:
These may well show up in the UI - for a user's history.

How about "entry predates audit records for user observations" or the like?